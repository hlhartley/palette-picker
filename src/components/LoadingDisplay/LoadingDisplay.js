import React from 'react';
// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

export const LoadingDisplay = () => {
    return (
        <div className="LoadingDisplay">
          <h2 className="title">Loading...</h2>
        </div>
    )
}

export default LoadingDisplay;