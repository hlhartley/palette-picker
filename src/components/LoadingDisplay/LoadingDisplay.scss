.LoadingDisplay {
    background-color: #e6f5f7;
}