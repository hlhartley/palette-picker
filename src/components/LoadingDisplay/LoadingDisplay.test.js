import React from 'react';
import { shallow } from 'enzyme';
import { LoadingDisplay } from './LoadingDisplay';

describe('LoadingDisplay', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(
            <LoadingDisplay/>
        )
    });

    describe('LoadingDisplay component', () => {
        it('should properly render the component elements', () => {
            expect(wrapper).toMatchSnapshot();
        });
    });
});