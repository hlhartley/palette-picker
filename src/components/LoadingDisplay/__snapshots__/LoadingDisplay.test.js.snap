// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingDisplay LoadingDisplay component should properly render the component elements 1`] = `
<div
  className="LoadingDisplay"
>
  <h2
    className="title"
  >
    <FontAwesomeIcon
      border={false}
      className="pencil-icon"
      fixedWidth={false}
      flip={null}
      icon={
        Object {
          "icon": Array [
            512,
            512,
            Array [],
            "f303",
            "M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z",
          ],
          "iconName": "pencil-alt",
          "prefix": "fas",
        }
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size={null}
      spin={false}
      symbol={false}
      title=""
      transform={null}
    />
     Loading...
  </h2>
</div>
`;
